src/components/Header/Header.scss

```diff
@@ -0,0 +1,49 @@
+.sa-header {
+  background-color: var(--sa-header-background-color);
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+
+  .header-wrapper {
+    display: flex;
+    justify-content: space-between;
+    position: relative;
+    font-size: 15px;
+    flex-direction: row;
+    text-align: center;
+    align-items: center;
+
+    .logo {
+      padding: 5px 5px 5px 5px;
+      margin-left: 3%;
+      .brand {
+        text-decoration: none;
+        display: flex;
+
+        img {
+          width: 35px;
+          height: auto;
+        }
+
+        .brand-name {
+          display: flex;
+          color: #000;
+          @include typographyMediumL;
```

Comment:
This one is typographyBoldXl in the design, you can check it by ctrl + clicking on the text, and you should see it in the sidebar on the right